apps/admin_api/test/admin_api/v1/controllers/self_controller_test.exs

```diff
@@ -1,8 +1,8 @@
 defmodule AdminAPI.V1.SelfControllerTest do
   use AdminAPI.ConnCase, async: true
-  alias EWalletDB.Account
+  alias EWallet.Helper
```

Comment:
Where are those changes coming from? `develop`? Was this branch of from `develop`? In that case, we might want to merge it there first?

Reply:
Welp, wrong base